.travis.yml

```diff
@@ -2,4 +2,9 @@ sudo: false
 language: ruby
 rvm:
   - 2.2.4
+sudo: false
```

Comment:
It looks like there's already a `sudo: false` at line 1, so I don't think you need this one. See the [online validator](https://lint.travis-ci.org/) to check.